src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -373,19 +383,21 @@ public function deleteWorkspace($name)
         try {
             $this->getConnection()->delete('phpcr_workspaces', ['name' => $name]);
         } catch (Exception $e) {
-            throw new RepositoryException("Couldn't delete workspace '$name': " .$e->getMessage(), 0, $e);
+            throw new RepositoryException("Couldn't delete workspace '$name': " . $e->getMessage(), 0, $e);
         }
 
         try {
             $this->getConnection()->delete('phpcr_nodes', ['workspace_name' => $name]);
         } catch (Exception $e) {
-            throw new RepositoryException("Couldn't delete nodes in workspace '$name': ".$e->getMessage(), 0, $e);
+            throw new RepositoryException("Couldn't delete nodes in workspace '$name': " . $e->getMessage(), 0, $e);
```

Comment:
the codestyles i usually use do not put spaces around the concatenation `.`. please revert those changes to keep the diff more focussed on what this PR is doing.

i am happy to have a separate PR that does more strict codestyle (ideally with styleci configuration to keep code style strict in the future)